src/main/java/org/janelia/saalfeldlab/fx/MenuFromHandlers.java

```diff
@@ -149,12 +149,11 @@ public static void main(String[] args)
 	{
 		PlatformImpl.startup(() -> {});
 
-		ContextMenu menu = new MenuFromHandlers(Arrays.asList(
```

Comment:
The `main` method is obsolete (was used to test functionality). Please remove the entire method in this PR.

Reply:
Done